administrator/language/de-DE/com_scheduler.ini

```diff
@@ -110,6 +110,7 @@ COM_SCHEDULER_SCHEDULER_TASK_ROUTINE_NA="Aufgabe#%1$02d Aufgabe hat keine entspr
 COM_SCHEDULER_SCHEDULER_TASK_START="Laufende Aufgabe#%1$02d '%2$s'."
 COM_SCHEDULER_SCHEDULER_TASK_UNKNOWN_EXIT="Aufgabe%1$02d wurde beendet mit Code %4$d in %2$.2f (net %3$.2f) Sekunden."
 COM_SCHEDULER_SCHEDULER_TASK_UNLOCKED="Aufgabe#%1$02d wurde entsperrt."
+COM_SCHEDULER_SCHEDULER_TASK_WILL_RESUME="Aufgabe#%1$02d benötigt mehr Arbeitszeit."
```

Comment:
@joomlagerman/reviewer please check this translation idea.

Reply:
```suggestion
COM_SCHEDULER_SCHEDULER_TASK_WILL_RESUME="Aufgabe#%1$02d benötigt mehr Bearbeitungszeit."
```
Als Idee?